### ISSUE TITLE
Create error screen for user not on mainnet

### ISSUE THREAD

**Issue by rachelhamlin** [status-im/ens-usernames#22]
The DApp will be launched publicly on mainnet. Any user who interacts with it on a testnet, like Ropsten, will be unable to search.

Use the styling below to present a warning message that says,

`Please connect to Ethereum mainnet to continue.`

![image](https://user-images.githubusercontent.com/15470435/46202761-f791ed00-c317-11e8-98db-f372acf53201.png)

This will replace the current error message for the testnet version of the DApp: 

![image](https://user-images.githubusercontent.com/15470435/46202869-3d4eb580-c318-11e8-96d3-e6ae305f6f99.png)

**Comment by bgits** [comment]
It's already there, when we deploy it to mainnet, Ropsten will be replaced with mainnet.

**Comment by bgits** [comment]
The functionality, not the new design

**Comment by rachelhamlin** [comment]
@bgits wdyt about borrowing this other error screen for the design?

I know the copy isn't super specific—e.g. if I'm not in a DApp browser at all I won't understand—but we can address that later as well

**Comment by bgits** [comment]
We already have an error screen for when the user is not on the correct network. It's used in two dapps, go to vote.status.im while connected to any network but mainnet. Is this purely an aesthetics issue?

**Comment by rachelhamlin** [comment]
Yup, I think it makes sense to have consistent styling across all screens. From what I remember the other error screen doesn't fit mobile either.

**Comment by bgits** [comment]
So maybe this issue should be called, redesign error screen? In terms of priorities where do you see this?

**Comment by rachelhamlin** [comment]
Sure thing, and pretty low. Leo or even Alex might tackle? Will update when
I’m at my computer later.
